src/AzureFunctions.PowerShell.OpenTelemetry.SDK/Logging/NewFunctionsOpenTelemetryLogger.cs

```diff
@@ -0,0 +1,34 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Management.Automation;
+using OpenTelemetryEngine.Logging;
+using OpenTelemetryEngine.ResponseObjects;
+
+namespace AzureFunctions.PowerShell.OpenTelemetry.SDK
+{
+    /// <summary>
+    /// Cmdlet that sets up the OpenTelemetry Logger
+    /// This Logger is configured with an OTLP exporter that will send logs to an OTLP endpoint defined by the user
+    /// See https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "FunctionsOpenTelemetryLogger")]
+    [OutputType(typeof(FunctionsLoggerResponse))]
+    public class NewFunctionsOpenTelemetryLogger : PSCmdlet
+    {
+        protected override void ProcessRecord()
+        {
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable(SDKConstants.FunctionsOpenTelemetryEnvironmentVariableName))) 
+            {
+                WriteError(new ErrorRecord(new InvalidOperationException("OpenTelemetry environment variable not set, logs from your function app's default logging pipeline will not be sent to OpenTelemetry")
```

Comment:
Please keep in mind that `WriteError` may or may not throw an exception depending on the current error action preference. In case it does not, does it make sense to proceed with the rest of the code (building/writing a response)? Should the rest of the code be in `else`?

Reply:
I have updated all WriteError calls from the missing env var checks to WriteWarning, which I feel is more appropriate for this case. Yes, in my opinion we should still continue the execution. The specifics of what will happen if the environment variable is not set: 

1. User logs emitted through the "typical" PowerShell Functions logging paths will not be emitted from the worker through this module - instead, they will flow to the host which will log them to OTel due to the lack of the worker capability
2. User logs emitted through the Write-FunctionsOpenTelemetryLog cmdlet will be sent to OTel by the module, but will not be correlated with the host telemetry.
3. User traces generated with Start-FunctionsOpenTelemetrySpan/Stop-FunctionsOpenTelemetrySpan will not be linked to host telemetry
4. Every call to New-FunctionsOpenTelemetryTracerBuilder, New-FunctionsOpenTelemetryLogger, Write-FunctionsOpenTelemetryLog, or Start-FunctionsOpenTelemetrySpan will emit a log message that will be sent with log level information back to host, and forwarded that way to OTel